src/cli/commands/dev.ts

```diff
@@ -73,6 +73,14 @@ export class Dev implements Command {
       dev: {
         addToWatcherSettings: {},
         async onBeforeWatcherStartOrRestart(change) {
+          if (change.type === 'change') {
+            const nexusModules = Layout.findNexusModules(
+              layout.tsConfig,
+              layout.app.exists ? layout.app.path : null
+            )
+            layout.update({ nexusModules })
```

Comment:
Whenever a file changes, we update the layout, **but only the nexus modules**, not the whole layout